projects/assets/src/translations/en/product.json

```diff
@@ -98,7 +98,7 @@
       "deliveryTab": "Shipping",
       "SparePartsTabComponent": " Spare Parts"
     },
-    "tabPanelContainerRegion": "Tab group with more product details"
+    "tabPanelContainerRegion": "Group with more product details"
```

Comment:
This is a breaking change. One suggestion is to create a new key and use that instead.
https://sap.github.io/spartacus-docs/breaking-changes/#avoiding-breaking-changes